### ISSUE TITLE
Xanthogranulomatous pyelonephritis

### ISSUE THREAD

**Issue by RichardKlaus** [obophenotype/human-phenotype-ontology#6861]
**Preferred term label:**
Xanthogranulomatous pyelonephritis

**Synonyms**

**Definition (free text, please give PubMed ID)**
Xanthogranulomatous pyelonephritis is a granulomatous inflammatory infiltrate composed of neutrophils, lymphocytes, plasma cells, xanthomatous histiocytes, and multinucleated giant cells.
PMID: 21526966

**Comment**
Xanthogranulomatous pyelonephritis is an uncommon chronic destructive granulomatous process of renal parenchyma in association with long-term urinary tract obstruction and infection. 

**Parent term (use hpo.jax.org/app)**
Pyelonephritis HP:0012330

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-2234-4248

**Comment by pnrobinson** [comment]
added
